### PR TITLE
IDEMPIERE-5695 - Process audit doesn't set print language, format, print format on Forms/documents

### PR DESCRIPTION
**Pull request by PeterTakacs300**
JIRA ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-5695

# Pull Request Checklist

- [x] My code follows the [code guidelines](https://wiki.idempiere.org/en/Contributing_to_Trunk) of this project
- [x] My code follows the best practices of this project
- [x] I have performed a self-review of my own code
- [x] My code is easy to understand and review. 
- [x] In hard-to-understand areas, I have commented my code.
- [x] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [ ] I have made corresponding changes to the documentation as follows:
  - [ ] New feature (non-breaking change which adds functionality): I have created the New Feature page in the project wiki explaining the functionality and how to use it. If relevant, I have committed sample data to the core seed to have usable examples in GardenWorld.
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected): I have documented the change in a clear way that everyone in the community can understand the impact of the change.
  - [ ] Improvement (improves and existing functionality): This documentation is needed if the improvement changes the way the user interacts with the system or the outcome of a process/task changes. If it is just, for instance, a performance improvement, documentation might not be needed. 
- [ ] The changed/added documentation is in the project wiki (not privately-hosted pdf files or links pointing to a company website) and is complete and self-explanatory.

### CODE REVIEW

File: org.adempiere.ui/src/org/compiere/print/ReportCtl.java [hunk 1]
```diff
@@ -397,7 +416,16 @@ public static boolean startDocumentPrint (int type, MPrintFormat customPrintForm
...
+			
+			// Set Print Format and Language to AD_Pinstance
+			if(instance.getAD_PrintFormat_ID() <= 0 && format.getAD_PrintFormat_ID() > 0) {
+				instance.setAD_PrintFormat_ID(format.getAD_PrintFormat_ID());
+				instance.saveEx();
+			}
+			if(instance.getAD_Language_ID() <= 0 && format.getLanguage() != null) {
+				instance.setAD_Language_ID(MLanguage.get(Env.getCtx(), format.getLanguage()).getAD_Language_ID());
+				instance.saveEx();
+			}
```
Comment (hengsin) [inline org.adempiere.ui/src/org/compiere/print/ReportCtl.java:428]: please add null check for instance here

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui/src/org/compiere/print/ReportCtl.java [hunk 2]
```diff
@@ -148,25 +149,25 @@ else if(instance.getAD_PrintFormat_ID() > 0)
...
+				return startDocumentPrint(ReportEngine.RFQ, pi.getRecord_ID(), parent, WindowNo, !pi.isPrintPreview(), instance);
 			else if (pi.getAD_Process_ID() == PROCESS_RPT_C_PAYMENT)		//	C_Payment
-				return startCheckPrint(pi.getRecord_ID(), !pi.isPrintPreview());
+				return startCheckPrint(pi.getRecord_ID(), !pi.isPrintPreview(), instance);
 			else if (pi.getAD_Process_ID() == PROCESS_RPT_M_INVENTORY)		//	Physical Inventory
-				return startDocumentPrint(ReportEngine.INVENTORY, pi.getRecord_ID(), parent, WindowNo, !pi.isPrintPreview());
+				return startDocumentPrint(ReportEngine.INVENTORY, pi.getRecord_ID(), parent, WindowNo, !pi.isPrintPreview(), instance);
 			else if (pi.getAD_Process_ID() == PROCESS_RPT_M_MOVEMENT)		//	Inventory Move
-				return startDocumentPrint(ReportEngine.MOVEMENT, pi.getRecord_ID(), parent, WindowNo, !pi.isPrintPreview());
+				return startDocumentPrint(ReportEngine.MOVEMENT, pi.getRecord_ID(), parent, WindowNo, !pi.isPrintPreview(), instance);
```
Comment (hengsin) [inline org.adempiere.ui/src/org/compiere/print/ReportCtl.java:170]: Should apply the same changes to startFinReport and startStandardReport too.

Comment (PeterTakacs300) [reply org.adempiere.ui/src/org/compiere/print/ReportCtl.java:169]: done

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**